openapi/openapi.yml

```diff
@@ -1087,6 +1102,47 @@ paths:
           $ref: '#/components/responses/Error'
       tags:
         - Private Subnet
+  '/project/{project_id}/location/{location}/private-subnet/{private_subnet_name}/connect':
+    parameters:
+      - $ref: '#/components/parameters/project_id'
+      - $ref: '#/components/parameters/location'
+      - $ref: '#/components/parameters/private_subnet_name'
+    post:
+      operationId: connectPrivateSubnet
+      summary: Connect private subnet to another private subnet
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                connected-subnet-ubid:
+                  type: string
+              additionalProperties: false
+      responses:
+        '200':
+          $ref: '#/components/responses/PrivateSubnet'
+        default:
+          $ref: '#/components/responses/Error'
+      tags:
+        - Private Subnet
+  '/project/{project_id}/location/{location}/private-subnet/{private_subnet_name}/disconnect/{private_subnet_id}':
```

Comment:
For consistency, I think maybe the private_subnet_id should have a _ prefix?

Reply:
I think that would actually break it, since the route doesn't accept the underscore.  The underscore is not necessary for disambiguation as you cannot use a name in this case, only a ubid.